packages/manager/apps/hycu/src/pages/onboarding/index.scss

```diff
@@ -0,0 +1,10 @@
+.tile-grid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-gap: 30px;
```

Comment:
Not rem for gap ?

Reply:
seems that file is not used. It should be removed from the generator rather